kernel/src/common/result.rs

```diff
@@ -0,0 +1,18 @@
+//! result.rs -- Standard return type for invoking operations, returning
+//! success or an error code.
+//!
+//!  Author: Philip Levis <pal@cs.stanford.edu>
+//!  Date: Nov 21, 2016
+
+enum Result {
+    SUCCESS,
+    FAIL, // Generic failure condition
+    EBUSY, // Underlying system is busy; retry
+    EALREADY, // The state requested is already set
+    EOFF, // The component is powered down
```

Comment:
Sometimes yes, but not always. E.g., if the RF233 is in DEEP_SLEEP, then trying to set the channel shouldn't turn the radio on.